### DISCUSSION TITLE
Toggle on the mining slowdown while flying

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#563] [Feature Requests]
When using creative flight in survival, with essentials or another plugin, or using an elytra (though, how you'd be mining while using an elytra idk) it slows your pick down. In order to instant-mine stone you need efficiency 13 in that situation with haste II, rather than efficiency 5. 
In a server like mine where most people have access to /fly it would be really nice to have that as a toggle, rather than unable to be turned off (unless it already is somewhere and I just don't know how)

TBH not sure if this is the place to ask for this feature or if I'd have to ask in essentials or something tbh but yeah

**Comment by gecko10000** [comment]
The only way I can see to do this is to automatically give the player +400% mining speed (since the slowdown when flying is 1/5) equivalent to Haste 16.

**Comment by mibby** [comment]
Seconding this request. Would be nice to turn off the flight encumbrance without having to resort to using Haste 16 since you may want to remain vanilla-esque but allow slightly easier building when hovering in the air.
